x/deployment/types/types.go

```diff
@@ -12,13 +12,18 @@ import (
 
 type attributesMatching map[string]types.Attributes
 
-// DefaultOrderBiddingDuration is the default time limit for an Order being active.
-// After the duration, the Order is automatically closed.
-// ( 24(hr) * 3600(seconds per hour) ) / 7s-Block
-const DefaultOrderBiddingDuration = int64(12342)
+const (
+	// ManifestVersionLength is the length of manifest version
+	ManifestVersionLength = 32
```

Comment:
It'd be nice to define this in the same place that the hash is calculated - `sdl/sdl.go`.  Are there import cycle issues if you do this?

If nothing else, you can use `sha256.Size`

Reply:
Yes, there are import cycle issues if we add in `sdl/sdl.go`. That's the reason, I moved it to types.